howtos/process/docbuild.rst

```diff
@@ -136,7 +136,7 @@ Depending on your Linux version, install the needed tools:
   .. code-block:: bash
 
      sudo apt-get install doxygen python3-pip python3-wheel make \
-        default-jre graphviz
+        default-jre graphviz python3-sphinx python3-breathe
```

Comment:
Hmmm... further down on the page the sphinx and breathe packages are installed using:

    pip3 install --user -r scripts/requirements.txt

Are you finding that's not working for you?

Reply:
I did that that and still saw error about missing Python packages. If the others are OK without this I can just close this PR.